test/testdata/deployednettemplates/recipes/scenario4/README.txt

```diff
@@ -0,0 +1,6 @@
+This configuration creates a network of 6 participating nodes and 30
+relays, such that the relays are distributed in 3 regions, and the
+nodes are in a 4th region.
+
+This scenario was designed to stress-test the situation where the
+agreement votes arrive late from different rounds. 
```

Comment:
`agreement votes arrive late from different rounds` -> `agreement proposal arrives for the next round.`